packages/snippet-bot/src/snippet-bot.ts

```diff
@@ -301,14 +299,10 @@ async function scanPullRequest(
         tagsFound = true;
       }
     } catch (e) {
-      const err = e as RequestError & Error;
-      // Ignoring 403/404 errors.
-      if (err.status === 403 || err.status === 404) {
-        logger.info(
-          `ignoring 403/404 errors upon fetching ${file}: ${err.message}`
-        );
+      if (e instanceof FileNotFoundError) {
```

Comment:
403s will now be fatal? Do we know why were ignoring 403s previously?

Reply:
I don't remember, but probably a mistake :)